### PR TITLE
Add client button renamed

### PR DESCRIPTION
**Pull request by ankitgaur07**
Targets #2279

### Description
Rename the add client button

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Resources/views/index.blade.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <div class="d-none d-md-flex justify-content-between my-2">
         @include('client::menu_header')
         @can('clients.create')
-            <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add client</a>
+            <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add new Client</a>
```
Comment (gautam071) [inline Modules/Client/Resources/views/index.blade.php:8]:
```suggestion
            <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add New Client</a>
```

**Review by gautam071 — APPROVED** [review]
Looks Good one minor chnage

File: Modules/Client/Resources/views/index.blade.php (hunk 1 above)
Comment (ankitgaur07) [reply Modules/Client/Resources/views/index.blade.php:8]: ok sir I will make suitable changes

**Review by Ayush8923 — APPROVED** [review]
Looks good.

**Pull request merged**